plugin.php

```diff
@@ -265,4 +264,65 @@ function update_duplicate_check_warning_option()
 	}
 });
 
+add_action('parse_request', function () use ( $pDI ) {
+	if ( str_contains($_SERVER["REQUEST_URI"], "onoffice-clear-cache") ) {
+		$pDI->get(CacheHandler::class)->clear();
+		$location = ! empty($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : admin_url('admin.php?page=onoffice-settings');
+		wp_safe_redirect($location);
+		exit();
+	}
+});
+
+if ( ! is_admin() ) {
+	add_action('admin_bar_menu', function ( $wp_admin_bar ) {
+		$toolBarConfig = [
+			[
+				'id'    => 'onoffice',
+				'title' => 'onOffice',
```

Comment:
I think the titles need to be translated, right?

Reply:
I added the translation, please check the PR for me!

Reply:
I added the translation, please check the PR for me!